.idea/misc.xml

```diff
@@ -3,7 +3,7 @@
   <component name="ProjectKey">
     <option name="state" value="project://e2804f05-5315-4fc6-a121-c522a6c26470" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="16" project-jdk-type="JavaSDK">
```

Comment:
If `languageLevel` is JDK_15, shouldn't `project-jdk-name` be 15 instead of 16?

Reply:
Again have a look at the file, the "-" at the line start means we are removing the line.

Here is what I changed it to:
`<component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">`